src/storage.tf

```diff
@@ -0,0 +1,86 @@
+resource "azurerm_resource_group" "rg_storage" {
+  name     = format("%s-storage-rg", local.project)
+  location = var.location
+
+  tags = var.tags
+}
+
+module "storage_account" {
+  source = "./modules/storage_account"
+
+  name            = replace(format("%s-sa", local.project), "-", "")
```

Comment:
the sa suffix stands for? I mean what's the purpose of this storage?

Reply:
sa: storage_account.
We need this storage to keep all media objects of CMS (images, pdfs, etc.)

Reply:
can we rename it something like assets?